shuup/front/template_helpers/general.py

```diff
@@ -151,3 +153,12 @@ def get_pagination_variables(context, objects, limit):
     variables["objects"] = page.object_list
 
     return variables
+
+
+@contextfunction
+def get_price_info(context, product, quantity=1):
```

Comment:
Also the filters should be used instead of get_price_info. We should avoid the performance issue for those (like proper caching).